server/app/views/applicant/UpsellParams.java

```diff
@@ -38,6 +38,9 @@ public static Builder builder() {
   // Will be non-empty if programTitle is used
   abstract Optional<String> programDescription();
 
+  // Program ID of the program that was just applied to
+  abstract long programId();
```

Comment:
**suggestion (non-blocking):** Give this a more descriptive name, since it's not obvious what programId would be associated with UpsellParams? Or maybe it is, just a thought. Maybe "completedProgramId".